tools/SetupFlow/DevHome.SetupFlow/ViewModels/PackageCatalogListViewModel.cs

```diff
@@ -138,6 +139,7 @@ private void RemoveShimmers(int count)
     [RelayCommand]
     private void ViewAllPackages(PackageCatalogViewModel catalog)
     {
+        TelemetryFactory.Get<ITelemetry>().LogCritical("Apps_ViewAll_Event");

```

Comment:
>Apps_ViewAll_Event [](http://example.com/codeflow?start=56&length=18)

Is there a particular event naming system being used in the codebase? 

(Without knowing what those rules are, I'd have otherwise suggested a name like "ViewAllAppsClicked" or something along those lines.)

Reply:
They seem to have a x_y format
![image](https://github.com/microsoft/devhome/assets/22826005/2e3acfe6-a1eb-4d7a-a25c-b1f379f7b303)